django/src/ft_auth/templates/settings.html

```diff
@@ -0,0 +1,160 @@
+{% extends "./extends/base.html" %}
+
+{% block content %}
+
+<div class="profile-container">
+	<div class="profile-header">
+		<div class="profile-image-wrapper">
+			<div class="profile-image-container">
+				<img src="{{ user.avatar.url }}" alt="User Profile Picture">
+			</div>
+			<div class="edit-icon">
+				<button id="edit-avatar-button" class="edit-button">
+					<i class="fas fa-pen"></i>
+				</button>
+			</div>
+		</div>
+	</div>
+	<hr class="profile-divider">
+	<div class="profile-content">
+		<div class="username-container">
+			<span class="username">{{ user.username }}</span>
+		</div>
+		<hr class="profile-divider">
+		<form id="password-form" data-url="/api/auth/password/update/" class="form-container">
+			<div class="row text-center text-wrap text-danger">
+				<p id="password-error"></p>
+			</div>
+			<div class="form-floating row">
+				<input type="password" class="form-control password-container" id="current_password"
+					placeholder="Current password" value="">
+				<label for="current_password">Current Password</label>
+			</div>
+			<div class="form-floating row">
+				<input type="password" class="form-control password-container" id="new_password"
+					placeholder="New password" value="">
+				<label for="new_password">New Password</label>
+			</div>
+			<div class="form-floating row">
+				<input type="password" class="form-control password-container" id="confirm_password"
+					placeholder="Confirm new password" value="">
+				<label for="confirm_password">Confirm New Password</label>
+			</div>
+			<button id="password-btn" class="confirm-btn">
+				Confirm changes
+			</button>
+		</form>
+	</div>
+</div>
+
+<div id="avatarModal" class="modal-overlay" style="display:none;">
+	<div class="modal-content">
+		<button type="button" class="close-button" id="closeModalBtn">&times;</button>
+		<h2 class="form-title">Upload New Avatar</h2>
+		<hr class="profile-divider">
+		<form id="avatarForm" enctype="multipart/form-data" class="avatar-form">
+			{% csrf_token %}
+			<input type="file" name="avatar" id="avatarInput" accept="image/*" required style="display: none;">
+			<label for="avatarInput" class="custom-file-label">Choose File</label>
+			<span id="file-name" class="file-name">No file chosen</span>
+			<small>Max file size: 2MB</small>
+			<button id="upload-avatar-btn" type="submit" class="upload-btn">Upload</button>
+		</form>
+	</div>
+</div>
+
+<script>
+	function showModal() {
+		const modal = document.getElementById('avatarModal');
+		modal.style.display = 'block';
```

Comment:
use class "hidden"

Reply:
adressed in : [c237b06](https://github.com/TheSecretOrganization/ft_transcendence/pull/247/commits/c237b0684f5f5df9034f7881b86370e17513c291)